content/applications/finance/accounting/bank/feeds/bank_statements.rst

```diff
@@ -1,72 +1,61 @@
+
 ===============
 Bank statements
 ===============
 
-Importing your bank statements in Odoo Accounting allows you to keep track of the financial
-movements that occur on your bank accounts and reconcile them with the transactions recorded in your
-accounting.
-
-We recommend you use bank synchronization for more efficiency. Please read the related
-documentation: :doc:`bank_synchronization`.
+Importing your bank statements allows you to keep track of your bank account transactions and
+reconcile them with the ones recorded in Odoo.
 
-However, if you don't want to use bank synchronization or if your bank is not a supported
-institution, you still have other options:
+We recommend using :doc:`the bank synchronization feature <bank_synchronization>` for more
+efficiency. However, if you do not want to use it or if your bank is not yet supported, other
+options exist:
 
-#. Import the bank statement files delivered by your bank
-#. Register the bank statements manually
+- Import the bank statement files delivered by your bank;
+- Register the bank statements manually.
 
 Import bank statements files
 ============================
 
 Odoo supports multiple file formats to import bank statements:
 
-- SEPA recommended Cash Management format (CAMT.053)
-- Comma-separated values (.CSV)
-- Open Financial Exchange (.OFX)
-- Quicken Interchange Format (.QIF)
-- Belgium: Coded Statement of Account (.CODA)
+- SEPA recommended Cash Management format (CAMT.053);
+- Comma-separated values (.CSV);
+- Open Financial Exchange (.OFX);
+- Quicken Interchange Format (.QIF);
+- Belgium: Coded Statement of Account (.CODA).
 
-To import them, go to :menuselection:`Accounting --> Overview --> Bank`, click on *Import
-Statements*, or on the three dots, and then on *Import Statement*.
+To import a file, go to the Accounting dashboard and click on :guilabel:`Import (Statements)` under
+the bank card, or click on the drop-down menu ⋮ button and then on :guilabel:`Import Statement`.
```

Comment:
```suggestion
To import a file, go to the Accounting dashboard and click on :guilabel:`Import (Statements)` under
the :guilabel:`Bank` card, or click on the drop-down menu button (:guilabel:`⋮`) and then on :guilabel:`Import Statement`.
```
- I would guilabel Bank as it appears on the UI and we refer to the specific UI element.
- I wonder if adding the word 'card' is clear for the average user (I know, it's my suggestion). Technically, it's a Kanban card, but maybe there is a better word for it.
- drop-down menu ⋮ button -> drop-down menu button (:guilabel:`⋮`). Now you can remove the brackets if you prefer, but I would put the vertical ellipsis character at the end. And guilabeling it as it's a UI element